_documentation/user/flags/seer.md

```diff
@@ -5,4 +5,14 @@ type: Good
 summary: See stealthed, cloaked and masquerading tanks as normal.
 ---
 
-...
+The Seer (SE) flag allows the player to see the tanks holding the [Stealth (ST)](../stealth/), [Cloaking (CL)](../cloaking/), or [Masquerade (MQ)](../masquerade/) flags as normal.
+
+Its power is therefore to nullify the effects of those three flags for the holder, rather than bringing out a unique power-up itself.
+
+## Tactics
+
+As the flag disables the effect of three other specific flags for the holder, the usage of Seer is highly situational.
+
+In general, a player would prefer a flag that gives a power-up itself rather than nullify that of some others, as a power-up can effect most players on the map, while Seer does not add any effect when battling against tanks that have flags other than the three affected by Seer.
+
+As such, the usage is usually limited to the situations where either a holder of one of the three affected flags is in close proximity to the player holding Seer, or the majority of the tanks on the map are holding one of the three affected flags.
```

Comment:
I'd opt towards talking about how Seer can be useful when hunting down rabbits with stealth or pesky users always shooting you in the back. I like the mention of this flag being powerful on maps with a lot of stealth.

Reply:
yeah ... great idea!

Reply:
@allejo @jwmelto thank you for the comments. While there can be a mention of finding stealth rabbits, I thought it would be redundant, as that isn't any more major than a teammate locating a stealth tank incoming towards the base on CTF and letting everyone on the team know, or a teammate (or even often times an enemy) locating a stealth hiding in a dangerous location on FFA and letting everyone on the map know.

We can mention these cases, but as they more or less span across all game modes, I thought it wouldn't make sense to have preferences towards one.

Reply:
The Seer flag doesn't introduce special effects like other flags, it instead nullifies the effect of another flag. For this reason, I think it'd be beneficial to discuss which tactics this flag can defeat since that's essentially the power of this flag. Rabbits using stealth is a fairly common one, a stealth-mostly map is another, a stealth camping in a base, etc.

Reply:
As said, we can list all. It might end up quite extensive, but it is better than discriminating some situations over others.